File: opportunity_job_api/src/apps/jobs/models/faq.py
from django.db import models
from django.utils.translation import gettext_lazy as _

from apps.common.models import BaseModel


class FAQ(BaseModel):
    class Meta:
        verbose_name = _("FAQ")
        verbose_name_plural = _("Frequently Asked Questions")

    question = models.CharField(
        verbose_name=_("question"),
        max_length=250,
    )
    answer = models.TextField(
        verbose_name=_("answer"),
    )
    display_to_anonymous = models.BooleanField(
        verbose_name=_("display to anonymous"),
        default=False
    )
    display_to_employers = models.BooleanField(
        verbose_name=_("display to employers"),
        default=False
    )
    display_to_applicants = models.BooleanField(
        verbose_name=_("display to applicants"),
        default=False
    )

File: opportunity_job_app/src/api/models/Faq.ts
export interface Faq{
    id: number,
    created: string,
    modified: string,
    question: string,
    question_en: string,
    question_sr_latn: string,
    answer: string,
    answer_en: string,
    answer_sr_latn: string,
    display_to_anonymous: boolean,
    display_to_employers: boolean,
    display_to_applicants: boolean
}